src/grpc.ts

```diff
@@ -291,7 +291,7 @@ export class GrpcClient {
     return this._getCredentials(options).then(credentials => {
       const grpcOptions: {[index: string]: {}} = {};
       Object.keys(options).forEach(key => {
-        if (key.indexOf('grpc.') === 0) {
+        if (key.indexOf('grpc.') === 0 || key.indexOf('grpcgcp.') === 0) {
```

Comment:
I don't get it now :)  As we found out yesterday, gRPC silently ignores options passed with `grpc.` prefix. Looks like I'm still missing something here.